src/Integration.Vsix/Suppression/SuppressionManager.cs

```diff
@@ -67,7 +67,7 @@ private void SetupSuppressionHandling()
         {
             liveIssueFactory = new LiveIssueFactory(serviceProvider);
             delegateInjector = new DelegateInjector(ShouldIssueBeReported, serviceProvider);
-            sonarqubeIssueProvider = new SonarQubeIssuesProvider(sonarQubeService, activeSolutionBoundTracker, timerFactory);
+            sonarqubeIssueProvider = new SonarQubeIssuesProvider(sonarQubeService, this.activeSolutionBoundTracker.ProjectKey, timerFactory);
```

Comment:
In this method, the suppression manager is handling the solution changed event and is creating a new issue provider.